junitparser/junitparser.py

```diff
@@ -551,6 +594,10 @@ def system_err(self, value):
             err = SystemErr(value)
             self.append(err)
 
+    @property
+    def is_success(self):
```

Comment:
Should not add this. Success can be subjective. Some may not consider `Skipped` as success.

Reply:
That's a good point, I'll add a skipped, passed, and failed property instead. It will cover my use case (and be even more readable).